http-api/src/main/java/io/avaje/http/api/IgnoreBeanField.java

```diff
@@ -0,0 +1,12 @@
+package io.avaje.http.api;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/** Mark a field on a BeanParam/FormParam class as not a request parameter of any kind */
+@Target(FIELD)
+@Retention(SOURCE)
+public @interface IgnoreBeanField {}
```

Comment:
I think this should just be called `Ignore`.

I realise that is a kind of name clash with `@Json.Ignore` but I think the changes of a "form bean" or "query params bean" also being a `@Json` type is relatively low.

In the case of a clash people can use `@Json.Ignore @Ignore private String ignored`